docs/vlab/demo.md

```diff
@@ -192,6 +192,50 @@ core@server-02 ~ $ ip a
        valid_lft forever preferred_lft forever
 ```
 
+> **_NOTE_** Make usre that route table on servers is properly configured if you setting up interfaces manually.
```

Comment:
typo: usre -> sure and can you describe potentially most popular problem about default routes